src/assets/PythonCopyFileWithShutil/python-copy-paste.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Ways to copy file with shutil module in Python
+authorName: Salipa Gurung
+authorAvatar: https://avatars.githubusercontent.com/u/53458341?v=4
+authorLink: https://github.com/Salipa-Gurung
+createdAt: Feb 27, 2024
+tags: python, copy-paste, shutil
+banner: https://blog.jankaritech.com/src/assets/PythonCopyFileWithShutil/images/python_copy_file.png
+---
+
+Python provides many built-in modules to copy and paste a file. We'll look into ways to copy file using `shutil` module in this blog post.
```

Comment:
As a reader I would be interested why shutil is recommended by you since you mention that there are many modules available for copy and pasting of files. Maybe mention one or two things that are particularly nice about shutil